sql/hive/src/test/scala/org/apache/spark/sql/hive/orc/OrcSourceSuite.scala

```diff
@@ -19,6 +19,7 @@ package org.apache.spark.sql.hive.orc
 
 import java.io.File
 
+import org.apache.orc.OrcConf.COMPRESS
 import org.scalatest.BeforeAndAfterAll
 
```

Comment:
Another test in `OrcSourceSuite`:

https://github.com/apache/spark/blob/d8f45408635d4fccac557cb1e877dfe9267fb326/sql/hive/src/test/scala/org/apache/spark/sql/hive/orc/OrcSourceSuite.scala#L153

Reply:
This is case sensitive test case. We should not change this.

Reply:
Oh, I was thinking something like `COMPRESS.getAttribute.toUpperCase`.

Reply:
If you want, no problem at all. :)

Reply:
Just like to be consistent. Thanks. :)

Reply:
Yep!